app/src/main/java/org/oppia/android/app/devoptions/featureflags/FeatureFlagsViewModel.kt

```diff
@@ -53,6 +75,9 @@ class FeatureFlagsViewModel @Inject constructor(
           featureFlagId = ephemeralFeatureFlag.id,
           currentValue = ephemeralFeatureFlag.currentValue,
           syncStatus = ephemeralFeatureFlag.syncStatus,
+          afterResetValue = ephemeralFeatureFlag.nonOverriddenValue,
+          afterResetSyncStatus = ephemeralFeatureFlag.nonOverriddenSyncStatus,
```

Comment:
Can we use `nonOverriddenValue` and `nonOverriddenSyncStatus` everywhere?
```suggestion
          nonOverriddenValue = ephemeralFeatureFlag.nonOverriddenValue,
          nonOverriddenSyncStatus = ephemeralFeatureFlag.nonOverriddenSyncStatus,
```
<br>

**Rationale:**

- `afterReset` suggests these are only available after pressing reset, but in fact they’re injected up front and always represent the canonical non-overridden value.
- It also sounds like an event outcome, while they’re really baseline state.

Would like to know your thoughts if you have any preference?

Reply:
Agree with these points, updated

Reply:
Thanks, also please verify if you have updated them in `PlatformParameterViewModel` as well?

Current `PlatformParameterViewModel`:
```kt
afterResetValue = ephemeralPlatformParameter.nonOverriddenValue,
afterResetSyncStatus = ephemeralPlatformParameter.nonOverriddenSyncStatus,
```

Reply:
updated for `PlatformParameterViewModel` as well.